src/components/styleElements/icons/Check.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
```

Comment:
I needed a Check and a X icon for the red and green ShotsButton component.  That's what these two icons are for.